selfdrive/car/toyota/values.py

```diff
@@ -958,16 +958,17 @@ class CAR:
       b'\x028966342V1000\x00\x00\x00\x00897CF1202001\x00\x00\x00\x00',
       b'\x028966342Y8000\x00\x00\x00\x00897CF1201001\x00\x00\x00\x00',
       b'\x02896634A18000\x00\x00\x00\x00897CF1201001\x00\x00\x00\x00',
+      b'\x02896634A43000\x00\x00\x00\x00897CF4201001\x00\x00\x00\x00',
     ],
     (Ecu.esp, 0x7b0, None): [
-      b'F152642520\x00\x00\x00\x00\x00\x00',
```

Comment:
This had to have been a wrong copy - paste at some point.. No ESP fw on non-Hybrids follow this convention.